pulsar-metadata/src/main/java/org/apache/pulsar/metadata/bookkeeper/BKCluster.java

```diff
@@ -197,13 +197,13 @@ protected void cleanupTempDirs() throws Exception {
         }
     }
 
-    private ServerConfiguration newServerConfiguration() throws Exception {
+    private ServerConfiguration newServerConfiguration(int index) throws Exception {
         File dataDir;
         if (clusterConf.dataDir != null) {
             dataDir = new File(clusterConf.dataDir);
         } else {
             // Use temp dir and clean it up later
-            dataDir = createTempDir("bookie", "test");
+            dataDir = createTempDir("bookie", index + "test");
```

Comment:
`"test" + index` seems better or `"test-" + index`.

Reply:
Ok, I updated it